packages/js/data/src/product-variations/index.ts

```diff
@@ -29,8 +24,12 @@ export const store = createCrudDataStore<
 	namespace: WC_PRODUCT_VARIATIONS_NAMESPACE,
 	storeConfig: {
 		reducer: reducer as Reducer< ResourceState >,
-		actions,
-		selectors,
+		actions: {
+			...actions,
+		} as ProductVariationActions,
+		selectors: {
+			...selectors,
+		} as ProductVariationSelectors,
```

Comment:
```suggestion
		actions: actions as ProductVariationActions,
		selectors: selectors as ProductVariationSelectors,
```
Re-spreading seems unnecessary here.

Reply:
Fixed with cebfac7f615cbe00efd5aa86077b733748b373f6